cli-main.js

```diff
@@ -31,6 +31,7 @@ const cli = meow(`
 	  --cwd=<dir>       Working directory for files
 	  --stdin           Validate/fix code from stdin
 	  --stdin-filename  Specify a filename for the --stdin option
+	  --print-config    Print the ESLint configuration for the given file
```

Comment:
```suggestion
	  --print-config    Print the effective ESLint config for the given file
```

Alternatively, "effective" => "resolved"